### ISSUE TITLE
Consider removing EPPlus depedency because of non permissive licence.

### ISSUE THREAD

**Issue by inomatical** [fullstackhero/blazor-starter-kit#330]
EPPlus has a [Polyform Noncommercial License 1.0.0](https://www.nuget.org/packages/EPPlus/5.7.0/license) which makes it a restrictive license for commercial use. Also this license conflicts with this project's **MIT License**. 

This can be confusing for users of this template and unwillingly and unknowingly infringe EPPlus license.

Please make a clear note of this, or remove altogether this dependency.

**Comment by vip32** [comment]
are there any 'non retrictive' alternatives? have looked but came no further that to stick with v4 of epplus

**Comment by fretje** [comment]
I personally have no experience with it, but [NPOI ](https://github.com/nissl-lab/npoi) seems to be well maintained and has an [Apache 2.0 licence](https://github.com/nissl-lab/npoi/blob/master/LICENSE).

**Comment by inomatical** [comment]
Yes there are.
* [ClosedXml](https://www.nuget.org/packages/ClosedXML/)
* [OpenXml](https://www.nuget.org/packages/DocumentFormat.OpenXml/)

EPPlus depends on OpenXml. So I think there are good alternatives.

**Comment by asilmertkan** [comment]
EPPlus V5 has a Polyform Noncommercial License 1.0.0 but V4 has a (last stabil version 4.5.3.3) GNU Library General Public License (LGPL). This projects  depency can be downgrade to V4. 

Project link for those who are interested https://github.com/JanKallman/EPPlus

**Comment by neozhu** [comment]
@umunu I agree with you, there is my code [ExcelService.cs](https://github.com/neozhu/RazorPageCleanArchitecture/blob/main/src/Infrastructure/Services/ExcelService.cs)

**Comment by hades200082** [comment]
I'd question why something like EPPlus or any Excel library is needed in a boilerplate clean architecture project in the first place?

If a BH dev user is building a project that needs those features they can add their own library of choice. Why bloat the boilerplate with it?
